packages/abap-deploy-config-inquirer/src/prompts/conditions.ts

```diff
@@ -225,6 +225,9 @@ export function defaultOrShowSearchPackageQuestion(
     useAutocomplete = false
 ): boolean {
     // Only show the autocomplete prompt when the autocomplete prompt is supported; CLI or YUI specific version
```

Comment:
"YUI specific version" is no longer relevant as its been supported since 1.10.something.

Reply:
Updated based on conversation.